_plugins/edit.md

```diff
@@ -15,3 +15,11 @@ info:
 A simple admin for Pico with pages/404/options editor.
 
 Pico Edit provides a back-end interface to edit Pico pages. Additionally, it has the ability to perform some basic Git operations such as commit, push/pull etc.
+
+**Features**
+
+* Simple and clean interface
```

Comment:
Yup.  It's on my to-do list :point_up:

If you've got a quick fix for it, go ahead.  I haven't investigated it any further yet. :wink: